### PR TITLE
fix: 🐛 total label alignment and inner radius slider

### PR DESCRIPTION
**Pull request by apryka**
Closes: #116

### CODE REVIEW

File: packages/charts/src/charts/donut/donut-total.component.tsx [hunk 1]
```diff
@@ -12,32 +12,34 @@ const DonutTotal: FC<Props> = memo(({ children, ...typography }) => {
   const { fontSize, fontFamily } = rest;
   const totalLabelSize = 14;
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/donut/donut-total.component.tsx]: Why it's 14 not for example 22 or 13 ? :)

Comment (apryka) [reply packages/charts/src/charts/donut/donut-total.component.tsx]: font size is taken from UI https://projects.invisionapp.com/d/#/console/19037699/396877841/preview

Comment (apryka) [reply packages/charts/src/charts/donut/donut-total.component.tsx]: should it be editable as other labels?

Comment (maciejrybaniec) [reply packages/charts/src/charts/donut/donut-total.component.tsx]: IMO - yes as hard coding this value here could be a misleading in future.

File: packages/charts/src/charts/donut/donut-chart.component.tsx [hunk 2]
```diff
@@ -173,8 +178,13 @@ export const DonutChart: FC<Props> = ({
               />
             )
           )}
-          {donutSettings.total?.enabled && (
-            <DonutTotal {...donutSettings.total.typography}>
+          {totalEnabled && (
+            <DonutTotal
+              total={{
+                label: { ...donutTotalLabel.typography },
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/donut/donut-chart.component.tsx]: Why using `spread` here ? Looks like it accepts 1:1 Typography interface.

File: packages/charts/src/charts/donut/donut-total.component.tsx [hunk 3]
```diff
@@ -1,44 +1,81 @@
...
+      </tspan>
+    </text>
   );
+  // const { fontColor, ...rest } = typography;
+  // const { fontSize, fontFamily } = rest;
+  // const totalLabelSize = 14;
+  // return (
+  //   <text
+  //     y={`-${totalLabelSize / 2}`}
+  //     pointerEvents="none"
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/donut/donut-total.component.tsx]: Is this still required ?

File: packages/charts/src/charts/donut/donut-total.component.tsx [hunk 4]
```diff
@@ -1,43 +1,47 @@
...
-        pointerEvents="none"
+    <text
+      y={`-${label.fontSize / 2}`}
+      pointerEvents="none"
+      style={{
+        textAnchor: 'middle',
+        dominantBaseline: 'middle',
+      }}
+    >
+      <tspan
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/donut/donut-total.component.tsx]: Is `<tspan>` a valid SVG element ? Why not used `<text>` here ?

Comment (apryka) [reply packages/charts/src/charts/donut/donut-total.component.tsx]: `tspan` is valid, here are the [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/tspan)

I use `<text>` to group label/value and display it in the middle, then add a vertical offset to substring in `<tspan>`

File: packages/charts/src/types.ts [hunk 5]
```diff
@@ -112,7 +112,12 @@ export type Donut = {
   };
   total: {
     enabled: boolean;
-    typography: Typography;
+    label: {
```
Comment (maciejrybaniec) [inline packages/charts/src/types.ts:115]: Small thing - could we move `typography` at `total` property level ?

**Pull request merged**